doc/configcommands.dsv

```diff
@@ -80,7 +80,7 @@ newsblur-passwordeval||<command>||""||Another secure alternative, is providing y
 newsblur-url||<url>||"https://newsblur.com"||Configures the URL where the NewsBlur instance resides.||newsblur-url "https://localhost"
 notify-always||[yes/no]||no||If set to `no`, notifications will only be made when there are new feeds or articles. If set to `yes`, notifications will be made regardless.||notify-always yes
 notify-beep||[yes/no]||no||If set to `yes`, then the speaker will beep on new articles.||notify-beep yes
-notify-format||<string>||"newsboat: finished reload, %f unread feeds (%n unread articles total)"||Format string that is used for formatting notifications. See the chapter on format strings for more information.||notify-format "%d new articles (%n unread articles, %f unread feeds)"
+notify-format||<string>||"Newsboat: finished reload, %f unread feeds (%n unread articles total)"||Format string that is used for formatting notifications. See the chapter on format strings for more information.||notify-format "%d new articles (%n unread articles, %f unread feeds)"
```

Comment:
Please revert this change: the third column here is the default value of the setting, and since this PR doesn't (and shouldn't) change the default, it shouldn't change this line too.